### PR TITLE
Prep for 4.3.1 release

### PR DESCRIPTION
**Pull request by beylmk**
- Bumped purchases-android to 4.3.1 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.3.1), 
which fixes canMakePayments not returning (see related issue: https://github.com/RevenueCat/purchases-unity/issues/61)
- Bumped purchases-ios to 3.12.3 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.3)
- Bumped purchases-hybrid-common to 1.9.1 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/1.9.1)

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 1. Update to the latest SDK versions in `scripts/build.js`, `RNPurchases.podspec` and `android/build.gradle`.
 1. Update versions in VERSIONS.md.
 1. Update version in `package.json`.
-1. Update `platformFlavorVersion` in `RNPurchases.m` and `RNPurchasesModule.java`.
+1. Update `platformFlavorVersion` in `RNPurchases.m` and `PLUGIN_VERSION`in RNPurchasesModule.java`.
```
Comment (vegaro) [inline RELEASING.md:4]: Thanks for updating this!

Comment (vegaro) [reply RELEASING.md:4]: I think this step is missing tho. I don't see any updates to neither `RNPurchases.m` nor `RNPurchasesModule.java`.

**Review by vegaro — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 1 above)
Comment (beylmk) [reply RELEASING.md:4]: wowow. forgetting to press save in vscode has gotten me a few times now

Comment (vegaro) [reply RELEASING.md:4]: It happens. I activated the Autosave feature in the File menu because it also caused me trouble before

Comment (vegaro) [inline RELEASING.md:4]:
```suggestion
1. Update `platformFlavorVersion` in `RNPurchases.m` and `PLUGIN_VERSION` in RNPurchasesModule.java.
```

**Review by vegaro — APPROVED** [review]

**Pull request merged**